lib/v2/screens/profile_screens/guardians/guardians_tabs/guardians_screen.dart

```diff
@@ -212,3 +219,49 @@ void _showRemoveGuardianDialog(BuildContext buildContext, GuardianModel guardian
     },
   );
 }
+
+void _showOnboardingGuardianDialogSingleAction(ShowOnboardingGuardianSingleAction pageCommand, BuildContext buildContext) {
+  showDialog(
+      context: buildContext,
+      builder: (BuildContext context) {
+        return Center(
+          child: SingleChildScrollView(
+            child: OnboardingDialogSingleAction(
+                buttonTitle: pageCommand.buttonTitle,
+                indexDialong: pageCommand.index,
+                image: pageCommand.image,
+                description: pageCommand.description,
+                onNext: () {
+                  BlocProvider.of<GuardiansBloc>(buildContext).add(OnNextGuardianOnboardingTapped());
+                  Navigator.pop(context);
+                }),
+          ),
+        );
+      });
+}
+
+void _showOnboardingGuardianDialogDoubleAction(ShowOnboardingGuardianDoubleAction pageCommand, BuildContext buildContext) {
+  showDialog(
+      context: buildContext,
+      builder: (BuildContext context) {
+        return Center(
+          child: SingleChildScrollView(
+            child: OnboardingDialogDoubleAction(
+              rightButtonTitle: pageCommand.rightButtonTitle,
+              leftButtonTitle: pageCommand.leftButtonTitle,
+              indexDialong: pageCommand.index,
+              image: pageCommand.image,
+              description: pageCommand.description,
+              onNext: () {
+                BlocProvider.of<GuardiansBloc>(buildContext).add(OnNextGuardianOnboardingTapped());
+                Navigator.pop(context);
+              },
+              onPrevious: () {
+                BlocProvider.of<GuardiansBloc>(buildContext).add(OnPreviousGuardianOnboardingTapped());
+                Navigator.pop(context);
+              },
+            ),
+          ),
+        );
+      });
+}
```

Comment:
Widget components?

Reply:
@gguijarro-c-chwy , should I move this two into components?

Reply:
I think it's fine so I hadn't noticed that these are not methods they return to a widget so I guess they shouldn't have any drawback.